packages/typespec-python/src/code-model.ts

```diff
@@ -218,25 +218,28 @@ export function emitCodeModel<TServiceOperation extends SdkServiceOperation>(
         clients: [],
         subnamespaceToClients: {},
     };
+    for (const client of sdkPackage.clients) {
+        codeModel["clients"].push(emitClient(sdkContext, client));
+        if (client.nameSpace === sdkPackage.rootNamespace) {
+        } else {
+            codeModel["subnamespaceToClients"][client.nameSpace] = emitClient(sdkContext, client);
+        }
+    }
+    // loop through models and enums since there may be some orphaned models needs to be generated
     for (const model of sdkPackage.models) {
```

Comment:
Put loop for models/enums behind emitClients so that we could collect models which is not generate and disable them.